File: solutions/code360/42-square_root_of_a_number.py
def floorSqrt(n):
   # write your code logic here .
   low = 0
   high = n
   ans = 1

   while(low <= high):
      mid = (low + high) // 2
      if(mid * mid > n):
         high = mid - 1
      else:
         ans = mid
         low = mid + 1
   return ans

n= int(input())
print(floorSqrt(n))

File: solutions/code360/42-square_root_of_a_number.cpp
int floorSqrt(int n)
{
    int ans = 1;
    // Write your code here.
    for(int i = 1; i <= n; i++){
        if(i * i > n){
            break;
        }else{
            ans = i;
        }
    }
    return ans;
}
